bin/submit-task

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+## This script submits new tasks for SAPS interacting with Dispatcher backend.
+
+## Checking args
+if [ $# -ne 12 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD LOWER_LEFT_LATITUDE UPPER_RIGHT_LATITUDE LOWER_LEFT_LONGITUDE UPPER_RIGHT_LONGITUDE PROCESSING_INIT_DATE PROCESSING_FINAL_DATE INPUTDOWNLOADING_TAG PREPROCESSING_TAG PROCESSING_TAG DISPATCHER_ACCESS_IP"
```

Comment:
rename all references for DISPATCHER_ACCESS_IP to DISPATCHER_ADDRESS

Reply:
also, indicate the user should pass in the format IP:PORT, then remove the constant

Reply:
Done